src/DonationForms/resources/app/form/MultiStepForm/index.tsx

```diff
@@ -55,23 +55,23 @@ const convertSectionsToSteps = (sections: Section[], hasFirstStep: boolean) => {
 /**
  * @since 3.0.0
  */
-export default function MultiStepForm({sections, showHeader}: {sections: Section[]; showHeader?: boolean}) {
-    const steps = convertSectionsToSteps(sections, showHeader);
+export default function MultiStepForm({form}) {
```

Comment:
Let's make sure to add the correct `Form` type for this prop.

Reply:
@jonwaldstein we don't have `Form` type, right?

Reply:
We do, `import type {Form} from '@givewp/forms/types';`

Reply:
Resolved https://github.com/impress-org/givewp/pull/7059/commits/a66f19bdf4813abbdaf78d68d128f02020bdfcb1